### PR TITLE
Identity KotsKind CRD

### PR DESCRIPTION
**Pull request by salahalsaleh**

### CODE REVIEW

File: kotskinds/apis/kots/v1beta1/identity_types.go [hunk 1]
```diff
@@ -17,71 +17,45 @@ limitations under the License.
...
-	DisablePasswordAuth    bool              `json:"disablePasswordAuth,omitempty" yaml:"disablePasswordAuth,omitempty"`
-	Groups                 []IdentityGroup   `json:"groups,omitempty" yaml:"groups,omitempty"`
-	IngressConfig          IngressConfigSpec `json:"ingressConfig,omitempty" yaml:"ingressConfig,omitempty"`
-	AdminConsoleAddress    string            `json:"adminConsoleAddress,omitempty" yaml:"adminConsoleAddress,omitempty"`
-	IdentityServiceAddress string            `json:"identityServiceAddress,omitempty" yaml:"identityServiceAddress,omitempty"`
-	CACertPemBase64        string            `json:"caCertPemBase64,omitempty" yaml:"caCertPemBase64,omitempty"`
-	InsecureSkipTLSVerify  bool              `json:"insecureSkipTLSVerify,omitempty" yaml:"insecureSkipTLSVerify,omitempty"`
-	DexConnectors          DexConnectors     `json:"dexConnectors,omitempty" yaml:"dexConnectors,omitempty"`
+type IdentitySpec struct {
+	OIDCRedirectURLS            []string `json:"oidcRedirectURLS,omitempty" yaml:"oidcRedirectURLS,omitempty"`
```
Comment (emosbaugh) [inline kotskinds/apis/kots/v1beta1/identity_types.go:24]: i think this is required... maybe remove omitempty

**Review by emosbaugh — APPROVED** [review]

File: kotskinds/apis/kots/v1beta1/identity_types.go (hunk 1 above)
Comment (salahalsaleh) [reply kotskinds/apis/kots/v1beta1/identity_types.go:24]: good point

Comment (salahalsaleh) [reply kotskinds/apis/kots/v1beta1/identity_types.go:24]: actually, what if the vendor decides to not support oidc?

Comment (emosbaugh) [reply kotskinds/apis/kots/v1beta1/identity_types.go:24]: you mean not support the identity service? they will omit the doc entirely

**Review by emosbaugh — APPROVED** [review]

**Pull request merged**